clients/unshaded/src/main/java/tachyon/client/ClientContext.java

```diff
@@ -96,7 +96,7 @@ public static synchronized TachyonConf getConf() {
   }
 
   /**
-   * @return the ClientMetrics for this client
+   * @return the {@link ClientMetrics} for this client
```

Comment:
@HorizonNet , what's your thought of using relative path vs full path in `@link`?